doc/whats_new/v0.23.rst

```diff
@@ -180,72 +107,30 @@ Changelog
 
 - |Fix| :func: `cross_val_predict` supports `method="predict_proba"`
   when `y=None`.
-  :pr:`15918` by :user:`Luca Kubin <lkubin>`.
-
-:mod:`sklearn.naive_bayes`
-.............................
-
-- |Fix| A correctly formatted error message is shown in
-  :class:`naive_bayes.CategoricalNB` when the number of features in the input
-  differs between `predict` and `fit`.
-  :pr:`16090` by :user:`Madhura Jayaratne <madhuracj>`.
-
-:mod:`sklearn.neighbors`
-..............................
-
-- |Fix| Fix a bug which converted a list of arrays into a 2-D object 
-  array instead of a 1-D array containing NumPy arrays. This bug
-  was affecting :meth:`neighbors.NearestNeighbors.radius_neighbors`.
-  :pr:`16076` by :user:`Guillaume Lemaitre <glemaitre>` and  
-  :user:`Alex Shacked <alexshacked>`.
-
-:mod:`sklearn.neural_network`
-.............................
-
-- |Fix| Increases the numerical stability of the logistic loss function in
-  :class:`neural_network.MLPClassifier` by clipping the probabilities.
-  :pr:`16117` by `Thomas Fan`_.
+  :pr: `15918` by :user: `Luca Kubin <lkubin>`.
 
 :mod:`sklearn.preprocessing`
 ............................
 
 - |Efficiency| :class:`preprocessing.OneHotEncoder` is now faster at
   transforming. :pr:`15762` by `Thomas Fan`_.
 
-:mod:`sklearn.svm`
-..................
-
-- |API| :class:`svm.SVR` and :class:`svm.OneClassSVM` attributes, `probA_` and
-  `probB_`, are now deprecated as they were not useful. :pr:`15558` by
-  `Thomas Fan`_.
-
-- |Fix| Fix use of custom kernel not taking float entries such as string
-  kernels in :class:`svm.SVC` and :class:`svm.SVR`. Note that custom kennels
-  are now expected to validate their input where they previously received
-  valid numeric arrays.
-  :pr:`11296` by `Alexandre Gramfort`_ and  :user:`Georgi Peev <georgipeev>`.
-
 :mod:`sklearn.tree`
 ...................
 
 - |Fix| :func:`tree.plot_tree` `rotate` parameter was unused and has been
   deprecated.
   :pr:`15806` by :user:`Chiara Marmo <cmarmo>`.
 
-- |Fix| Fix support of read-only float32 array input in ``predict``,
-  ``decision_path`` and ``predict_proba`` methods of
-  :class:`tree.DecisionTreeClassifier`, :class:`tree.ExtraTreeClassifier` and
-  :class:`ensemble.GradientBoostingClassifier` as well as ``predict`` method of
-  :class:`tree.DecisionTreeRegressor`, :class:`tree.ExtraTreeRegressor`, and
-  :class:`ensemble.GradientBoostingRegressor`.
-  :pr:`16331` by :user:`Alexandre Batisse <batalex>`.
```

Comment:
The merge conflict went wrong somehow for the what's new. There should be no diff here aside for your changes. Will push a commit to fix it.

Reply:
Hi @rth ! The last lines in the .rst file I added are not there in the repo.

Reply:
@rth Thanks for accepting the pull request :)